features/src/main/feature/camel-features.xml

```diff
@@ -1617,6 +1618,7 @@
         <bundle>mvn:org.apache.camel.karaf/camel-javascript/${project.version}</bundle>
     </feature>
     <feature name='camel-jaxb' version='${project.version}' start-level='50'>
+        <feature prerequisite="true">spifly</feature>
```

Comment:
This was added to fix the warning from https://github.com/apache/camel-karaf/pull/515/files#r1782774809. 

After adding this the test works without the need for the changes mentioned here https://github.com/apache/camel-karaf/pull/515/files#r1781629096